chart-parts/ingress.yaml

```diff
@@ -1,51 +1,43 @@
-{{- if .Values.services.ingress -}}
+{{- if .Values.ingress.enabled -}}
 ---
-# The cert and the key cannot be filled automatically. If we could it
-# would roughly look like the lines below. As is it is the
-# responsibility of the operator to retrieve the values of these
-# secrets from the deployed SCF and update this secret with the actual
-# values. And the responsibility of the developer to supply the proper
-# instructions to the operator.
-##
-# tls.crt: {{ default "" .Values.secrets.ROUTER_SSL_CERT     | b64enc | quote }}
-# tls.key: {{ default "" .Values.secrets.ROUTER_SSL_CERT_KEY | b64enc | quote }}
```

Comment:
The comment needs to be updated, but still needs to live somewhere.

Reply:
I removed it here for the same reason I stated below. I don't think it belongs to the template.

Reply:
Brought a relevant comment back.